HRSPLT-425 msg former employees in Payroll Info to mitigate no self-serv

apetro opened the pull request:
In Payroll Information, mitigate former employee lack of self-service access to earnings statements and W-2s issued in 2019 (so, the W-2 for tax year 2018 that you might want to use over the next month or so to prepare your personal taxes).

Adds a message at the top of the portlet, showing on all tabs, to employees who lack the `UW_EMPLOYEE_ACTIVE` HRS role, which in practice seems to be mostly or entirely former employees.

Revised mockup, as implemented:

<img width="634" alt="mockup - Payroll Information - top message to former employees" src="https://user-images.githubusercontent.com/952283/54434761-f9504100-46fc-11e9-9c43-82146165b8f0.png">

apetro committed:
HRSPLT-425 msg former employees in Payroll Info to mitigate no self-serv

apetro committed:
changelog: link 182 Payroll Info employee no role mitigation

kedarjoyner commented:
@apetro This looks good! Have some minor style suggestions if time permits, but may just be something to consider in the future: 

- Using the yellow background (##FFE082) per the design system (woot!) to "highlight an element on a page or show focus". I do think the color change helps bring the eye more towards that important info. 
- Left-aligning the content when it's longer may help the user read more naturally
- Bold text instead of uppercase to signify importance

![highlight-content-myuw](https://user-images.githubusercontent.com/12139428/54374294-ccdcec00-464c-11e9-9645-53609192b53d.jpg)

apetro commented:
Good suggestions. 

I've backlogged the yellow background ([HRSPLT-427](https://jira.doit.wisc.edu/jira/browse/HRSPLT-427) )and left aligning ([HRSPLT-428](https://jira.doit.wisc.edu/jira/browse/HRSPLT-428)) since those seem more about defining styles used throughout these portlets than about style introduced in this changeset, so it'd probably be right to address that at the CSS layer and to bring all the similar content in the related apps forward together on this to maintain consistency. I agree what you've got in that mockup is immediately, obviously more attractive and I'd be excited about implementing it. 

The bold rather than all caps is a good call, I'll implement that here.

apetro committed:
strong rather than all-caps to emphasize the no-longer-uw-employee pred

apetro commented:
For history, here's the version Kedar was commenting on:

<img width="1273" alt="mockup - Payroll Information - top message to former employees" src="https://user-images.githubusercontent.com/952283/54370902-72d92800-4646-11e9-95b7-9979d74a5a1b.png">

apetro committed:
shorten message to former employees, linking KB article

as specified https://teams.microsoft.com/l/message/19:8a87a6e270094560898e5650a26cccea@thread.skype/1552596394027?tenantId=2ca68321-0eda-4908-88b2-424a8cb4b0f9&groupId=a3219f87-2fe2-4de8-a229-85b17b5af56b&parentMessageId=1552578844097&teamName=MyUW&channelName=Review&createdTime=1552596394027 ;
